Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -509,10 +513,54 @@ def results_to_incidents_datetime(response, last_fetch):
     return incidents, last_fetch
 
 
-def fetch_incidents():
+def get_indicators_command():
+    search, _ = get_indicators_search_scan()
+    limit = int(demisto.args().get('limit', FETCH_SIZE))
+    indicators_list = []
+    i = 0
+    for hit in search.scan():
+        ioc = results_to_indicator(hit)
+        if ioc:
+            indicators_list.append(ioc)
+        i += 1
+        if i >= limit:
+            break
+    hr = tableToMarkdown('Indicators', indicators_list, ['value'])
+    return_outputs(hr, {'ElasticsearchFeed.SharedIndicators': indicators_list}, indicators_list)
+
+
+def fetch_indicators_command():
+    search, now = get_indicators_search_scan()
+    for hit in search.scan():
+        ioc = results_to_indicator(hit)
+         if ioc:
+            demisto.createIndicators([ioc])
```

Comment:
@DeanArbel why don't we do this in batches?

Reply:
Also, please check that it works properly if we get indicator with the same value from two+ tenants

Reply:
Talked with you about this, you said that there shouldn't be a difference performance wise, so I did this as a quick win. If you think it's better to use batch I'll edit it

Reply:
I probably misunderstood, would be better in batches.